### PR TITLE
Update AGOL ETL to match changes in component view

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/15553

I also coded in an em dash (`—`) instead of a dash (`-`) in the SQL view, which was causing it's own problems 🤦 

## Testing
**Steps to test:**
Locally, you can follow the instructions in the ETL readme. I have tested this locally with prod data and it works 👍 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-etl/arcgis/components_to_agol.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def main():
     # lines and points must be stored in different layers in AGOL
     all_features = {"lines": [], "points": [], "combined": []}
 
+    logger.info("Building Esri feature objects...")
```
Comment (johnclary) [inline moped-etl/arcgis/components_to_agol.py:69]: this takes a little while, so i added logging

**Comment by johnclary** [comment]
thanks @mddilley for pointing out an oversight in this PR. i just pushed up changes to correctly capture the `project_list_view` updates as well 🙏

File: moped-database/views/project_list_view.sql [hunk 2]
```diff
@@ -89,7 +87,7 @@ AS WITH project_person_list_lookup AS (
     work_activities.task_order_names_short,
     work_activities.task_orders,
     'placeholder text' as project_development_status,
-    '2024-01-01T00:00:00−06:00' as project_development_status_date,
+    '2024-01-01T00:00:00-06:00' as project_development_status_date,
```
Comment (frankhereford) [inline moped-database/views/project_list_view.sql:90]: Woa, that's subtle! 😵‍💫

Comment (chiaberry) [reply moped-database/views/project_list_view.sql:90]: subtle for sure!!

**Review by mddilley — APPROVED** [review]
Built and tested this locally earlier today, and it ran like a dream! Thanks for patching 🙌 🚀 🚢

**Review by chiaberry — APPROVED** [review]
timed out once, but then was chugging along and worked 100% 


🚢

File: moped-database/views/project_list_view.sql (hunk 2 above)
Comment (johnclary) [reply moped-database/views/project_list_view.sql:90]: my IDE tried to warn me!

thing

**Comment by johnclary** [comment]
@chiaberry i hit a few timeouts as well. i'm surprised this doesn't fail more often on Airflow. in any case, we are very close to being able to switch to incremental updates thanks to DB trigger/audit fields 😅

**Pull request merged**